integration-tests/src/test/resources/proposed-assets-json.js

```diff
@@ -0,0 +1,112 @@
+var proposedAssetsJson = {
+    "assets": [
+    {
+        "schema": "stellar",
+        "code": "USDC",
+        "issuer": "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP",
+
+        //move to assets: config
+        // "distribution_account": "GBN4NNCDGJO4XW4KQU3CBIESUJWFVBUZPOKUZHT7W7WRB7CWOA7BXVQF",
```

Comment:
🤔  It doesn't look like this file is being used anywhere. Can we delete it?

Reply:
I am also not sure about this. @stfung77 Can you confirm? If it is not being used in the pipeline, we can remove it.

Reply:
I will remove it.